account_move_vendor_ac_noinv/views/account_move_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_move_form_autocomplete_button_purchase_inherit" model="ir.ui.view">
+        <field name="name">account.move.form (Union Autocomplete)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
```

Comment:
This inheritance is wrong: this field doesn't exist in this view, in my test database fails indeed. In fact, purchase addon adds it. Could you check?